packages/dso-toolkit/reference/render/alerts-validation.html

```diff
@@ -15,27 +15,25 @@ <h2>Sectiebericht met samenvatting validatiemeldingen</h2>
         <tr>
           <td>1 veld fout</td>
           <td class="plain-content">
-            <div class="alert alert-danger" role="alert">
-              <span class="sr-only">Fout:</span>
+            <dso-alert status="danger">
```

Comment:
De `.sr-only` zou niet weg moeten.. Ff checken met nieuwe reference file

Reply:
Toch wel: Dit is namelijk wel hoe het component aangeroepen gaat worden. Ik vermoed dat dit weer anders wordt zodra de markup routine rekening houdt met nested web components markup renderen. Voor nu is het OK